lib/tpl/dokuwiki/css/_recent.css

```diff
@@ -45,6 +45,23 @@
 .dokuwiki form.changes li span.user {
 }
 
+/*____________ size differences ____________*/
+
+.dokuwiki li .sizechange {
+    color: #fff;
```

Comment:
Unfortunately we cannot use predefined colours here as they would not be any of the guaranteed colour placeholders but one of the other ones (or related to them). All underscored CSS files can be used in other templates which might not have defined those colours.